tests/manage/pv_services/test_pvc_evict_ceph_clients.py

```diff
@@ -0,0 +1,119 @@
+import logging
+import pytest
+import random
+import json
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.framework.testlib import tier2
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import get_worker_nodes
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+class TestPvcEvictCephClients:
+    """
+    Test the ceph-fs PVC (RWX) mount with the following scenarios involving evict ceph-fs clients:
+    1. When two pods tries to mount the same PVC and scheduled on same node
+    2. When two pods tries to mount the same PVC and scheduled on different node
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["node"],
+        argvalues=[
+            pytest.param(
+                "same",
+                marks=[
+                    pytest.mark.bugzilla("1901499"),
+                    pytest.mark.polarion_id("OCS-3985"),
+                ],
+            ),
+            pytest.param(
+                "different",
+                marks=[pytest.mark.polarion_id("OCS-3984")],
+            ),
+        ],
+    )
+    def test_pvc_evict_ceph_clients(
+        self, node, pvc_factory, teardown_factory, pod_factory
```

Comment:
Where is it used the teardown factory. In fact you don't need it because pvc_factory and pod_factory have finalizers that do automatic teardown in the end of test or if it fails. Anyway your are not using it so remove it.